packages/airnode-adapter/src/response-processing/extraction.test.ts

```diff
@@ -295,14 +295,14 @@ describe('respects maximum encoded size', () => {
   });
 
   describe('encoding int256 arrays', () => {
-    const maxInt256ArrayLength = 30;
+    const maxInt256ArrayLength = 510;
```

Comment:
I presume this is correct but out of curiosity, can somebody explain the math behind it? :smile:

Reply:
Previously: `32 * 30 = 1024 - 64`
Now: `32 * 510 = 16384 - 64`

And my understanding for the reason to subtract 64 is:

```ts
const arrLen = ethers.utils.arrayify(ethers.utils.defaultAbiCoder.encode(['int256[]'], [[]])).byteLength;
console.log(arrLen);
```
Prints
```sh
64
```
Same as does:
```ts
const arrLen = ethers.utils.arrayify(ethers.utils.defaultAbiCoder.encode(['string'], [''])).byteLength;
```
hence:
https://github.com/api3dao/airnode/blob/5dd3b4a7dd387c413bee154c73f33f40511d14a8/packages/airnode-adapter/src/response-processing/extraction.test.ts#L278